tests/test_router_quote_out_given_in.py

```
@@ -287,3 +286,26 @@ def test_orderbook(self, environment_url, amount, token_pair):
+        denom_out = "uion"
+
+        expected_status_code = 200
+
+        # Fillbot address and slippage tolerance
+        # We choose fillbot address because we expect it to have at least one OSMO.
+        fillbot_address = "osmo10s3vlv40h64qs2p98yal9w0tpm4r30uyg6ceux"
+        # Note: relaxed
+        simulation_slippage_tolerance = 0.8
+
+                # Run the quote test
+        quote =  ExactAmountInQuote.run_quote_test(environment_url, token_in_coin, denom_out, False, True, EXPECTED_LATENCY_UPPER_BOUND_MS, expected_status_code, fillbot_address,  simulation_slippage_tolerance)
```

Ruff

`ExactAmountInQuote` may be undefined, or defined from star imports

(F405)